cluster/gce/gci/configure_helper_test.go

```diff
@@ -106,35 +106,38 @@ func (c *ManifestTestCase) mustCreateManifestDstDir() {
 	}
 }
 
-func (c *ManifestTestCase) mustCreateEnv(envTemplate string, env interface{}) string {
+func (c *ManifestTestCase) mustInvokeFunc(env interface{}, scriptName, targetTemplate string, templates ...string) {
+	envScriptPath := c.mustCreateEnv(env, targetTemplate, templates...)
+	args := fmt.Sprintf("source %s ; source %s; %s", envScriptPath, scriptName, c.manifestFuncName)
```

Comment:
use %q instead of %s

Reply:
@awly PTAl
The manifestFuncName needs to be %s, otherwise the test breaks.